pkg/console/operator/status-prefixes.go

```diff
@@ -0,0 +1,53 @@
+package operator
+
+// Requirements for ConditionPrefixes:
+// - A prefix MUST be single use.  We do not want to stomp on ourselves.
+// - IF a prefix is DEPRECATED, it MUST be registered with the
+//   NewRemoveStaleConditions() in pkg/console/starter/starter.go
+//   An orphaned condition could leave the operator in a permanently stuck state
+//   in the event of an upgrade.
+// - handlers MUST NOT be hidden behind IF checks.  a handler is responsible
+//   for toggling true||false on each pass.  If the handler is hidden behind a
+//   conditional, then the condition will be permanently set.
+
+// ConditionPrefix will be converted back to string when set as a Status.Condition
+// This type exits to help us ensure we are thoughtful about the conditions we use.
+type ConditionPrefix string
+
+// Available
+const (
+	// False means the deployment is not ready
+	ConsoleDeploymentReady ConditionPrefix = "ConsoleDeploymentReady"
+	// False means the deployment is not updated
+	ConsoleDeploymentUpdated ConditionPrefix = "ConsoleDeploymentUpdated"
+	// False means the route is not admitted
+	ConsoleRouteAdmitted ConditionPrefix = "ConsoleRouteAdmitted"
+)
+
+// Degraded
+const (
+	// True means the custom logo config is... probably invalid
+	ConsoleCustomLogoConfigmapSync ConditionPrefix = "ConsoleCustomLogoSync"
+)
+
+// Progressing & Degraded
+// HandleProgressingOrDegraded() will determine if this is a progressing or degraded error,
+// based on the type of error provided.
+const (
+	// True means something is wrong with route. what?
```

Comment:
Comments here point out how this is still not good enough.  We know now what component is broken instead of "its broken", but we still don't know why it's broken.